docs/data/reads.md

```diff
@@ -0,0 +1,39 @@
+---
+id: reads
+title: Reads
+sidebar_label: Reads
+---
+
+ICGC ARGO provides access to data from ARGO member programs generated through the standardized [ARGO Analysis pipelines](/docs/analysis_workflows/data-types). The following page lists genomic read analyses and data files generated by the ARGO Analysis pipelines.
+
+## Submitted Reads
+
+Raw reads submitted by ARGO member programs are used as inputs within the ARGO analytic pipeline. Data is accepted in FASTQ or BAM (aligned or unaligned) formats, must be compressed, and can be aligned or unaligned.
```

Comment:
should `aligned or unaligned` be repeated in this sentence?

Reply:
also..should "unaligned reads" be a section on this page too? or does it just not produce files?

Reply:
no unaligned reads are what are submitted to us. ARGO does not produce nor host unaligned reads.

Reply:
um the () is specifically that the BAM can be wither, where as FASTQ is only unaligned.

Reply:
this just seems redundant, but maybe I'm reading it wrong?

Data is accepted in FASTQ or BAM (`aligned or unaligned`) formats, must be compressed, and `can be aligned or unaligned`

Reply:
updated to `Data is accepted in FASTQ, unaligned BAM, or aligned BAM formats, and must be compressed for submissions.`

Reply:
yeah I think that's clearer...Could also write as

Data is accepted in FASTQ, unaligned/aligned BAM formats, and must be compressed for submissions.